packages/identity-kit/src/adapters/internet-identity/index.ts

```diff
@@ -24,11 +24,11 @@ export class II
 
   getSupportedStandards(): SupportedStandard[] {
     return [
-      { type: Standard.ICRC25, url: "url" },
-      { type: Standard.ICRC31, url: "url" },
-      { type: Standard.ICRC32, url: "url" },
-      { type: Standard.ICRC33, url: "url" },
-      { type: Standard.ICRC34, url: "url" },
+      { type: "ICRC_25", url: "url" },
```

Comment:
Do we really need the url field for the getSupportedStandards? Might it be better for us to have ICRC as enum and return only it? Or have type and url as a constant values. It has to be less likely that url and type can intersect between types and url for the same type could be different.